tests/js/pouchdb-component-tests/pouchdb-component-common-tests.js

```diff
@@ -0,0 +1,356 @@
+    inputs: {
+        bulkData: [
+            {
+                "_id": "one",
+                "saying": "is the loneliest number"
+            },
+            {
+                "_id": "two",
+                "saying": "are required to tango"
+            }
+        ],
+        mapReduce: [
+            {
+                "color": "red",
+                "name": "strawberry"
+            },
+            {
+                "color": "yellow",
+                "name": "banana"
+            },
+            {
+                "color": "red",
+                "name": "cherry"
+            },
+            {
+                "color": "blue",
+                "name": "blueberry"
+            },
+            {
+                name: "colorless glass noodles"
+            }
+        ]
+    },
+    expectedResponses: {
+        bulkCreate: [{ ok: true, id: "one"}, { ok: true, id: "two"}]
+    },
+    rawModules: [{
+        name: "Common tests for `gpii.pouch` component...",
+        type: "test",
+        tests: [
+            {
+                name: "Test the `info` method...",
+                sequence: [
+                    {
+                        func: "{testEnvironment}.pouchDb.info"
+                    },
+                    {
+                        event:    "{testEnvironment}.pouchDb.events.onInfoComplete",
+                        listener: "jqUnit.assertLeftHand",
+                        args:     ["The database info should be as expected...", { db_name: "test", doc_count: 0, update_seq: 0}, "{arguments}.0"]
+                    }
+                ]
+            },
+            {
+                name: "Test the `POST` method (with an ID)...",
+                sequence: [
+                    {
+                        func: "{testEnvironment}.pouchDb.post",
+                        args: [{ _id: "foo"}]
+                    },
+                    {
+                        event: "{testEnvironment}.pouchDb.events.onPostComplete",
+                        listener: "jqUnit.assertLeftHand",
+                        args: ["The POST should have been successful...", { ok: true, id: "foo"}, "{arguments}.0"]
+                    }
+                ]
+            },
+            {
+                name: "Test the `GET` method (with a non-existent ID)...",
+                sequence: [
+                    {
+                        func: "{testEnvironment}.pouchDb.get",
+                        args: [{ _id: "notGonnaFindIt"}]
+                    },
+                    {
+                        event: "{testEnvironment}.pouchDb.events.onError",
+                        listener: "jqUnit.assert",
+                        args: ["An error should be thrown if a record cannot be found..."]
+                    }
+                ]
+            },
+            {
+                name: "Test the `PUT` method (with an ID)...",
+                sequence: [
+                    {
+                        func: "{testEnvironment}.pouchDb.post",
```

Comment:
Hi, Cindy.  Can you somehow indicate that the change has been made?  I don't see an option to do anything about the request from this end.